media/css/teach/smarton.less

```diff
@@ -0,0 +1,1431 @@
+        letter-spacing: normal;
+    }
+
+    h4 {
+        .font-size(18px);
+    }
+
+    ul > li:before {
+        content: '';
+    }
+
+    .billboard {
+        padding: 0;
+        background: transparent;
+        border: 0;
+        box-shadow: none;
+    }
+
+    .form-contents,
+    .form-submit {
+        padding-top: .75em;
+    }
+
+    input[type='email'] {
+        .font-size(18px);
+        padding: .35em 20px;
+        border: 2px solid #eee;
+        border-radius: 5px;
+    }
+
+    .button {
+        background: transparent;
+        box-shadow: none;
+        color: #fff;
+        border: 2px solid #eee;
+        font-weight: normal;
+        text-shadow: none;
+        .font-size(18px);
+        padding: .35em 20px;
+
+        &:hover,
+        &:focus {
+            color: #fff;
+            border-color: #fff;
+            background-color: rgba(255, 255, 255, .05);
+        }
+    }
+
+    a {
+        color: #fff;
+    }
+
+    &.fixed {
+        position: fixed;
+        bottom: 0;
+        left: 0;
+        right: 0;
+        margin: 0;
+        padding: 0;
+        z-index: 99;
+    }
+
+    .close {
+        .image-replaced();
+        background: transparent;
+        border: 0;
+        cursor: pointer;
+        height: 30px;
+        width: 30px;
+        padding: 0;
+        position: absolute;
+        right: 20px;
+        top: 10px;
+
+        &:before {
+            .font-size(20px);
+            line-height: 30px;
+            font-weight: bold;
+            text-align: center;
+            vertical-align: middle;
+            text-indent: 0;
+            content: '\d7';
+            color: #fff;
+            display: block;
+            width: 30px;
+            height: 30px;
+            position: absolute;
+            left: 0;
+            top: 0;
+        }
+    }
+}
+
+body.not-firefox .footer-cta {
+    .foot-newsletter {
+        display: none;
+    }
+    .foot-download {
+        display: block;
+    }
+}
+
+.foot-download {
+    display: none;
+
+    .title,
+    .download-button,
+    .tagline {
+        display: table-cell;
+        vertical-align: middle;
+        padding: 0 10px;
+    }
+
+    .title {
+        width: 33%;
+    }
+
+    .tagline {
+        .open-sans-light();
+        .font-size(18px);
+        margin: 1em 0 0;
+    }
+}
+
+#smarton-foot-download .download-other {
+    display: none !important;
+}
+
+
+
+
+
+// Multi-column blocks
+.columns {
+    .flexbox;
+    margin: 0 auto;
+    padding: 0 @gridGutterWidth;
+
+    @media screen and (max-width: @breakTablet) {
+        padding: 0;
+    }
+
+    .col {
+        margin: 0 10px @baseLine;
+        .border-box;
+
+        ul {
+            list-style: disc;
+            margin: 0 0 1em;
+        }
+    }
+}
+
+.columns-2 .col { width: 50%; }
+.columns-3 .col { width: 33.3%; }
+.columns-4 .col { width: 25%; }
+.columns-5 .col { width: 20%; }
+
+
+
+@media screen and (max-height: 650px) {
+
+    // .nav-steps {
```

Comment:
I was still tinkering with this while working on mobile nav... it's meant to shave a few more pixels off the header in short viewports, but now that I'm fairly happy with then mobile nave I think I can uncomment it.